### ISSUE TITLE
Run tasks after incremental compilation is finished

### ISSUE THREAD

**Issue by PLEEROCK** [ivogabe/gulp-typescript#271]
I want to run some task / execute some code right after each compilation in incremental compilation is finished. In my case I want to restart my node application. What are the ways to do it?

**Comment by ivogabe** [comment]
You can listen to the `end` event like this:
```javascript
gulp.task('scripts', function() {
    return gulp.src(...)
        ...
        .on('end', function() {
            console.log('compilation finished');
        });
});
```
Or you can create a new task, that executes the code:
```javascript
gulp..task('scripts', function() { ... });
gulp.task('restart', ['scripts'], function() { ... });
```
With this approach, you should trigger the `restart` task instead of the `scripts` task. Gulp will automatically run `scripts` before `restart` starts.
